src/pages/OfferNew/Perils/PerilCollection/index.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from '@emotion/styled'
+import * as React from 'react'
+import { Peril } from '../types'
+import { PerilItem } from './PerilItem'
+
+interface Props {
+  perils: Peril[]
+  setCurrentPeril: (index: number) => void
+  setIsShowingPeril: (isShowingPeril: boolean) => void
+}
+
+const Wrapper = styled('div')`
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+`
+
+export const PerilCollection: React.FC<Props> = ({
```

Comment:
`React.FC<T>` 🤯

Stealing this for sure

Reply:
Yeah, it's very neat